data/data/config.tf

```diff
@@ -123,3 +123,14 @@ Should the cluster be created with ipv6 networking.
 EOF
 
 }
+
+variable "default_description" {
```

Comment:
This variable is declared in the root module but is not passed down to the other modules. Consequently, if the user does set this variable, it will only apply to resources created by the root module.

One of two changes should be made.
1) Make `default_description` a local value in each module rather than an input variable.
2) Pass `default_description` from the root module to each nested module. Do not provide a default value of the variable in the nested module to enforce that the caller of the module must supply the variable.

My preference is (1) as I do not think that we need to make `default_description` variable.

Reply:
I was hoping to create a global variable so in the future it would be easy to change the value but now since this is not working the way I intended, I think (1) would be easier. I'll make the changes.

Reply:
Terraform does not have a concept of global variables.